contrib/download-frozen-image-v2.sh

```diff
@@ -201,6 +201,58 @@ handle_single_manifest_v2() {
 	manifestJsonEntries=("${manifestJsonEntries[@]}" "$manifestJsonEntry")
 }
 
+get_target_arch() {
+	if [ -n "${TARGETARCH}" ]; then
```

Comment:
```suggestion
	if [ -n "${TARGETARCH:-}" ]; then
```

(to make sure it works if `TARGETARCH` is undefined and we `set -u`)

Reply:
Oh nice trick!